src/app/interfaces/concerts-upcoming-item.interface.ts

```diff
@@ -5,10 +5,11 @@ export interface ConcertUpcomingItem {
   readonly title: string;
   readonly date: string;
   readonly dateCode: string;
+  readonly concertLengthInMinutes: number;
   readonly place: string;
   readonly description: string;
   readonly inviteImageSource?: string;
   readonly mapUrl?: string;
-  readonly calendar?: string;
+  readonly calendarUrl?: string;

```

Comment:
tovabbra is opcionalis marad? nem lesz mindig megadva?

Reply:
calendarUrl így nem is lesz